components/home/menu/calendar/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+// import apiCalendar from 'react-google-calendar-api';
```

Comment:
Please remove commented import. Does not appear to be needed

Reply:
Has been fixed!